src/extension.ts

```diff
@@ -94,13 +94,46 @@ export async function activate(context: vscode.ExtensionContext) {
 		vscode.env.openExternal(uri);
 	});
 
+	let createCommand = vscode.commands.registerCommand('cargo-atcoder-vscode.new', async () => {
+		const contestIdentifier = await vscode.window.showInputBox({
+			"prompt": "AtCoder contest identifier (e.g. `abc100`): ",
+		})
+		if (contestIdentifier == null) {
+			vscode.window.showErrorMessage("Empty contest identifier.");
+			return;
+		}
+		vscode.window.showErrorMessage(contestIdentifier);
+		const configurations = vscode.workspace.getConfiguration("cargo-atcoder-vscode");
+		const rootDir = configurations.rootDir;
+
+		const task = createTask(
+			'new',
+			`cargo atcoder new ${contestIdentifier}`,
+			{cwd: rootDir},
+		);
+		vscode.tasks.executeTask(task);
+
+		vscode.tasks.onDidEndTask(e => {
+			// const openFolder = path.join(rootDir, contestIdentifier);
+			// vscode.window.showErrorMessage(openFolder);
+			// await vscode.commands.executeCommand('vscode.openFolder', openFolder);
+			vscode.window.showErrorMessage("raech");
+			vscode.commands.executeCommand(
+				'vscode.openFolder',
+				'/Users/himkt/Desktop/arc005',
+				true,
+			);
+		});
```

Comment:
To wait a `executeTask`, we need to use `onDidEndTask`.
https://stackoverflow.com/questions/61428928/how-to-await-a-build-task-in-a-vs-code-extension